scripts/relayCompiler.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv/config')
+
+const chalk = require('chalk')
+const fs = require('fs')
+const path = require('path')
+const spawn = require('child_process').spawn
+const { execSync } = require('child_process')
+
+ensureRequirementsMet()
+
+// prettier-ignore
+const args = [
+  '--extensions', 'js', 'jsx', 'ts', 'tsx',
+  '--schema', path.resolve(process.cwd(), './data/schema.graphql'),
+  '--language', 'typescript',
+  '--src', path.resolve(process.cwd(), '../'),
+  '--artifactDirectory', path.resolve(process.cwd(), 'src/__generated__'),
+  '--include',
+      'force/src/**',
+      'reaction/src/**',
+  '--exclude',
+      'reaction/node_modules/**',
+];
+
+if (process.argv.includes('--watch')) {
+  args.push('--watch')
+}
+
+// TODO: We don't need this as we're cloning projects side-by-side and setting
+// relay --src root one level below project, where reaction lives.
+
+// Use this flag if linking node modules
+// if (process.argv.includes('--no-watchman')) {
+//   args.push('--no-watchman')
+// }
+
+const proc = spawn(
+  path.resolve(__dirname, '../node_modules/.bin/relay-compiler'),
+  args,
+  { stdio: 'inherit' }
+)
+
+proc.on('close', code => {
+  process.exit(code)
+})
+
+function ensureRequirementsMet() {
```

Comment:
Nice, like this pattern!

Reply:
This file / process can go away once this is moved into Reaction, but yeah i was anticipating that we'd need a little more env setup than actually did